src/renderer/containers/App/index.tsx

```diff
@@ -62,6 +63,29 @@ message.config({
   maxCount: 1,
 });
 
+export function handleUploadJobUpdates(job: JSSJob, dispatch: Dispatch<any>){
+      // An FSS job update happens when:
```

Comment:
This is a bit pedantic so probably ignore, but I'd imagine the job changes in other scenarios too right? We just only care about the 3 you described.

Reply:
How about if i move the comment to inside the condition?
`      if (job.service === Service.FILE_STORAGE_SERVICE) {
`

Reply:
pushed